src/content/docs/it/concepts/islands.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Isole Astro
+description: "Le Isole Astro (noto anche come Isole Component) sono un modello di architettura web sviluppato da Astro. Il concetto di “architettura delle isole” è stato coniato per la prima volta dalla sviluppatrice frontend di Etsy, Katie Sylor-Miller, nel 2019, ed è stato ampliato dal creatore di Preact, Jason Miller."
+---
+
+import IslandsDiagram from '~/components/IslandsDiagram.astro';
+
+**Isole Astro** (noto anche come Isole di componenti) sono un modello di architettura web sviluppato da Astro. L'idea di "architettura delle isole" è stata coniata per la prima volta dalla sviluppatrice frontend di Etsy, [Katie Sylor-Miller](https://twitter.com/ksylor), nel 2019, ed è stata approfondita in [questo articolo](https://jasonformat.com/islands-architecture/) dal creatore di Preact, Jason Miller.
+
+## Cosa sono le Isole Astro?
+
+Il termine "Isola Astro" si riferisce a un componente UI interattivo all'interno di una pagina HTML statica. Su una pagina possono esistere più isole, e un'isola viene sempre renderizzata da sola. Pensate a esse come isole in un mare di HTML statico e non interattivo.
+
+<IslandsDiagram>
+  <Fragment slot="headerApp">Intestazione (isola interattiva)</Fragment>
+  <Fragment slot="sidebarApp">Barra laterale (HTML statico)</Fragment>
+  <Fragment slot="main">
+    Contenuto statico come testo, immagini, ecc.
+  </Fragment>
+  <Fragment slot="carouselApp">Carosello di immagini (isola interattiva)</Fragment>
+  <Fragment slot="footer">Piè di pagina (HTML statico)</Fragment>
+  <Fragment slot="source">Fonte: [Architettura delle isole: Jason Miller](https://jasonformat.com/islands-architecture/)</Fragment>
```

Comment:
I would not translate the title of an external source, so here I would keep "Island Architecture"

Reply:
I applied the fixes!